src/test.ts

```diff
@@ -19,7 +20,25 @@ function clearRequireCache() {
   });
 }
 
-const setup: SetupFn = ({ compare, getCtx, Given, setCtx, Then, When }) => {
+const setup: SetupFn = ({
+  compare,
+  getCtx,
+  Given,
+  setCtx,
+  Then,
+  When,
+  onTearDown,
+}) => {
+  // === App setup === //
+  Given('app setting "(.*)" is enabled', (setting) => {
```

Comment:
NITPICK: I would refer this step for enabling a setting for _a specific app_

```suggestion
  Given('the (advanced|basic) app setting "(.*)" is enabled', (setting) => {
```